src/LiveDevelopment/MultiBrowserImpl/transports/node/NodeSocketTransportDomain.js

```diff
@@ -58,7 +58,7 @@
     var _clients = {};
     
     // This must match the port declared in NodeSocketTransport.js.
-    // TODO: randomize this?
+    // Default socket port number
     var SOCKET_PORT = 8123;
```

Comment:
Since there is a pref for the socket port, this should be read from the prefs too.